src/__tests__/BookDetails.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { MemoryRouter as Router } from "react-router-dom"
+import { BookDetails } from '../containers/BookDetails';
+
+describe("A BookDetails component", function() {
+  it('fetches a book on mount', () => {
+    const fetchBook = jasmine.createSpy('fetchBook');
+    const wrapper = shallow(<BookDetails match={{ params: { isbn: 1 } }} fetchBook={fetchBook} />);
```

Comment:
I strongly recommend to use react-testing-library, it forces one to write tests in a better way and is much smaller than enzyme https://blog.kentcdodds.com/introducing-the-react-testing-library-e3a274307e65

Reply:
Cool, thanks for the hint. Changing to react-testing-library will need to rewrite the slides. Let's do this rewrite in a second step.